x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/XPackClientPlugin.java

```diff
@@ -568,16 +565,11 @@ public List<NamedWriteableRegistry.Entry> getNamedWriteables() {
                 // Data Tiers
                 new NamedWriteableRegistry.Entry(XPackFeatureSet.Usage.class, XPackField.DATA_TIERS, DataTiersFeatureSetUsage::new),
                 // Archive
-                new NamedWriteableRegistry.Entry(XPackFeatureSet.Usage.class, XPackField.ARCHIVE, ArchiveFeatureSetUsage::new)
+                new NamedWriteableRegistry.Entry(XPackFeatureSet.Usage.class, XPackField.ARCHIVE, ArchiveFeatureSetUsage::new),
+                // TSDB Downsampling / Rollup
+                new NamedWriteableRegistry.Entry(LifecycleAction.class, RollupILMAction.NAME, RollupILMAction::new)
```

Comment:
Downampling ILM is still work in progress and we don't want this to be made available before I merge PR https://github.com/elastic/elasticsearch/pull/87269

So, please remove it completely from here and I will enable it in #87269